### PR TITLE
Feat/civ 2898

### PR DESCRIPTION
**Pull request by bhagyashreesharma90**
Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/CIV-2898

### Change description ###
Added camunda scheduler for everyday 18:00hrs
Added camnunda process to send email notification for claimant and defendent solicitor in case of evidence upload in last 24 hours

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/resources/camunda/evidence_upload_scheduler.bpmn [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    <bpmn:sequenceFlow id="Flow_03at42s" sourceRef="StartEvent_1" targetRef="EVIDENCE_UPLOAD_CHECK" />
+    <bpmn:sequenceFlow id="Flow_1m5i8op" sourceRef="EVIDENCE_UPLOAD_CHECK" targetRef="Event_1ifnpoe" />
+    <bpmn:serviceTask id="EVIDENCE_UPLOAD_CHECK" name="Evidence Upload scheduler" camunda:type="external" camunda:topic="EVIDENCE_UPLOAD_CHECK">
+      <bpmn:incoming>Flow_03at42s</bpmn:incoming>
+      <bpmn:outgoing>Flow_1m5i8op</bpmn:outgoing>
+    </bpmn:serviceTask>
+    <bpmn:startEvent id="StartEvent_1">
+      <bpmn:outgoing>Flow_03at42s</bpmn:outgoing>
+      <bpmn:timerEventDefinition id="TimerEventDefinition_1oppfcm">
+        <bpmn:timeCycle xsi:type="bpmn:tFormalExpression">0 0 18 * * ?</bpmn:timeCycle>
```
Comment (m-meulendijks-v1) [inline src/main/resources/camunda/evidence_upload_scheduler.bpmn:13]: Have a look at TakeCaseOfflineScheduler + CaseDismissed scheduler.
They seem to run on the assumption that the working day stops at 4, and they run 4:05.
Did you discuss with someone that it should be 6?

**Review by m-meulendijks-v1 — DISMISSED** [review]

File: src/main/resources/camunda/evidence_upload_scheduler.bpmn (hunk 1 above)
Comment (bhagyashreesharma90) [reply src/main/resources/camunda/evidence_upload_scheduler.bpmn:13]: yes.. Doug asked me to put it at 6.

Comment (m-meulendijks-v1) [reply src/main/resources/camunda/evidence_upload_scheduler.bpmn:13]: Ok, no problem :)

**Review by hmcts-version1-ignacio — DISMISSED** [review]

**Review by drummondjm — APPROVED** [review]

**Comment by bhagyashreesharma90** [comment]
notification_camunda

**Review by nagaraju-deekonda — APPROVED** [review]

**Review by m-meulendijks-v1 — APPROVED** [review]

**Comment by filippo-possenti-version1** [comment]
Closing this to allow for a different branching strategy to avoid people getting blocked with changes from one another.

**Pull request closed without merging**